mosviz/loaders/loader_selection.py

```diff
@@ -194,12 +194,15 @@ def _validation_checks(self, *args, **kwargs):
             filenames = self.data.get_component(column_name).labels
 
             try:
-                loader(filenames[0])
-            except Exception as e:
+                path = os.sep.join(
+                    self.data._load_log.path.split(os.sep)[:-1])
+                file_path = os.path.join(path, filenames[0])
+                loader(file_path)
```

Comment:
I think the ``loader(file_path)`` should maybe be in its own try...except block?